doc/en/components/grids/_shared/column-types.md

```diff
@@ -237,41 +239,33 @@ The default template is using material icons for visualization of boolean values
 ```
 
 ```html
-<igc-column data-type="Boolean">
+<igc-column data-type="boolean">
 </igc-column>
 ```
 
-<!-- Blazor -->
-
 ### Image
 
-While there does not exist a built-in template for an image column in the `{ComponentName}` at this time, this can be simply implemented by providing a `BodyTemplateScript` to one of your `Column`s. This property points at a JavaScript function that can be defined like so:
+Default template is using the value coming from the data as an image source to a default image template. The default image template will extract the name of the image file and set it as `alt` attribute of the image to meet the accessibility requirement. The displayed cell size is adjusted to the sizes of the images rendered, so keep in mind that large images will still be rendered and the grid rows will become as large as the images in the image column. Filtering, sorting and grouping will be turned off by default for image type columns. If you want to enable them, you need to provide custom strategies which perform the data operations.
 
-```razor
-<IgbColumn Field="ImageSource" BodyTemplateScript="ImageCellTemplate" />
-
-//In JavaScript
-igRegisterScript("ImageCellTemplate", (ctx) => {
-    var html = window.igTemplating.html;
-    return html`<div>
-        <img src="${ctx.cell.value}"/>
-    </div>`;
-}, false);
+```html
+<igx-grid>
+    <igx-column [dataType]="'image'">
+    </igx-column>
+<igx-grid>
```

Comment:
why was blazor code snippet replaced with angular code snippet?

Reply:
Whole section is replaces since there's a dedicated image type now. You can run the docs and check that all sections show correct snippets.